### PR TITLE
refactor: replace "APNS_VOIP" with  "APNS" token

### PR DESCRIPTION
**Pull request by KaterinaWire**
## What's new in this PR?

This PR is a part of the Notification service extension refactoring. To be able to receive push notifications, we should register an "APNS" token.

### CODE REVIEW

File: Source/Push Token/MockTransportSession+PushToken.swift [hunk 1]
```diff
@@ -54,10 +54,14 @@ extension MockTransportSession {
     }
 
     func processPostPushToken(_ payload: ZMTransportData?) -> ZMTransportResponse {
+        var transportType = "APNS_VOIP"
+        if #available(iOS 13.0, *) {
+            transportType = "APNS"
+        }
```
Comment (johnxnguyen) [inline Source/Push Token/MockTransportSession+PushToken.swift:60]: This code will never run on user devices, but rather only on our testing machines. My guess is that they're all running iOS 13.

**Review by johnxnguyen — COMMENTED** [review]
So technically we should able to process both voip and standard push tokens. OS version checks aren't going to work here, so if you need to test processing both types of tokens you'll need another way to mock the OS version.

File: Source/Push Token/MockTransportSession+PushToken.swift (hunk 1 above)
Comment (KaterinaWire) [reply Source/Push Token/MockTransportSession+PushToken.swift]: Fixed

**Comment by KaterinaWire** [comment]
> So technically we should able to process both voip and standard push tokens. OS version checks aren't going to work here, so if you need to test processing both types of tokens you'll need another way to mock the OS version.

@johnxnguyen I will investigate if it's possible to mock an OS in our environment. For now, I removed the OS check. If I find a way how to do that, I will open another PR.

**Comment by johnxnguyen** [comment]
>  If I find a way how to do that, I will open another PR.

Can't you just add a flag to the mock transport session?

**Review by johnxnguyen — APPROVED** [review]

**Pull request merged**